eos-paygd/main.c

```diff
@@ -549,9 +551,12 @@ main (int   argc,
 
   allow_writing_to_boot_partition (FALSE);
 
+  /* If payg_sync_and_poweroff is scheduled, spin the mainloop until it runs */
+  while (timeout_id)
+      g_main_context_iteration (NULL, TRUE);
```

Comment:
The coding style for this project would either use a 2-space indent here, or have some `{` `}` on surrounding lines at that indent.

Reply:
I thought it was always a 4-space on lines containing code, and 2-space only before / after context delimiters (`{` `}`), but looking at [the spec](https://www.gnu.org/prep/standards/standards.html) you are actually right (I'm more of a K&R guy myself).

I'll fix this.